### PR TITLE
Lightweight text to image generation with aMUSED

### PR DESCRIPTION
**Pull request by aleksandr-mokrov**
CVS-129560

### CODE REVIEW

File: notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb [hunk 1]
```diff
@@ -0,0 +1,667 @@
+{
```
Comment (eaidova) [inline notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb:1]: <p>in review window this image looks very large and low quality. Can we reduce its size or remove?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (eaidova) [inline notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    %pip install "diffusers&gt;=0.25.0" gradio torch --extra-index-url https://download.pytorch.org/whl/cpu
</pre><p>openvino installation missed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (eaidova) [inline notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #9.    prompt = "cowboy"
</pre><p>can we test both pt and ov on the same prompt for making sure that results are the same?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (eaidova) [inline notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    
</pre><p>conv means converted? Maybe better use OV as prefix, I think it is more readable (Conv abbreviation usually used for convolution)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

**Comment by eaidova** [comment]
@aleksandr-mokrov  do you checked, is it possible to run notebook on colab?

**Review by eaidova — APPROVED** [review]

**Comment by aleksandr-mokrov** [comment]
> @aleksandr-mokrov do you checked, is it possible to run notebook on colab?

Added

File: notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb [hunk 2]
```diff
@@ -0,0 +1,689 @@
...
```
Comment (itrushkin) [inline notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    def generate(prompt, seed):
</pre><p>We can track the progress inside the Gradio interface using <code>gr.Progress</code> as the last function argument:</p><pre class="ql-syntax" spellcheck="false">def generate(prompt, seed, _=gr.Progress(track_tqdm=True))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

**Comment by itrushkin** [comment]
Notebook number can be set to 276.

**Comment by eaidova** [comment]
> Notebook number can be set to 276.

not actual anymore

File: notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb (hunk 2 above)
Comment (aleksandr-mokrov) [reply notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb:1]: Thanks, added

**Review by andrei-kochin — APPROVED** [review]

**Pull request merged**